packages/components/checkbox/stories/checkbox.stories.js

```diff
@@ -0,0 +1,15 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.checkbox';
+
+storiesOf('ts-checkbox', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const label = text('label', 'Label');
+		const disabled = boolean('Disabled', false);
+
```

Comment:
Do we need the value here ? 🤔 or maybe some actions to show the changed value?

Reply:
I don't think so. Most of the time developers don't need to change the value.